docs/software-platforms/lsfr/creating-vcs.md

```diff
@@ -0,0 +1,104 @@
+
+- The virtual computers provided through LSfR use an Ubuntu operating system.
+- During setup, you will choose your desired configuration, including the amount of virtual CPUs, memory, and storage space.
+    - You also have the option to add [additional storage to your virtual computer](STUB-LINK for volumes).
+- All virtual computers will have a set of pre-installed applications needed for working with OpenScPCA, including:
+    - Git and GitKraken
+    - The AWS command line interface
+    - R
+    - Conda
+
+!!! note "More on virtual computers"
+
+    Read more about [virtual computers with Lightsail for Research](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/computers.html).
+
+## How to create a virtual computer
+
+Follow the below steps to create a virtual computer to use with LSfR.
+
+1. Navigate to the [access portal URL from when you set up your user in IAM Identity center](../aws/index.md#joining-iam-identity-center).
+This link should bring you to the AWS Console and home page.
+
+    Once here, be sure that you are in the `us-east-2` region, by selecting the drop-down menu in the tool bar.
+
+    <figure markdown="span">
+        ![Select region](../../img/creating-vcs-1.png){width="400"}
+    </figure>
+
+1. You will need to use the AWS Service Catalog to create virtual computers.
+Open the AWS Service Catalog by using the search bar and typing, `service catalog`.
+
+    <figure markdown="span">
+        ![Select service catalog](../../img/creating-vcs-2.png){width="600"}
+    </figure>
+
+1. Select `LightsailInstance` in the product list, and click `Launch product`.
+
+    <figure markdown="span">
+        ![Launch product](../../img/creating-vcs-3.png){width="600"}
+    </figure>
+
+1. You will then choose the name and configurations for your virtual computer.
+
+    - Start by providing a `Provisioned product name`.
+    <!--TODO Do we want to provide guidance on names?-->
+    - Pick an `Application` to use from the drop-down menu.
+    This installs any additional applications along with the pre-installed applications.
+        - If you are planning to develop in R, we recommend choosing `Rstudio`, which includes RStudio.
+        - If you are planning to develop in Python, we recommend choosing `VSCodium`, which includes VSCodium, a distribution of Microsoft's editor VS Code.
+        - For information on all application options, see [the LSfR documentation on Applications](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/blueprints-plans.html).
+    - Choose any of the availability zones.
+    - Name your instance, be sure to make sure this is different than the name chose for your product.
+    - Pick the size for your instance.
+        - Use this [table describing the total vCPUs, memory, and storage space](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/blueprints-plans.html#plans) to choose the most appropriate instance plan.
```

Comment:
We need to recreate this table here to report pricing in `us-east-2`. See https://data-lab-knowledge.slab.com/posts/2024-03-23-jnt-notes-from-testing-researcher-aws-access-g0sdd98p#hnkio-information-only-section. I grabbed these from the LfR console. When you create via the LfR console, they will tell you how much things cost. I would also reiterate that they have a monthly budget and link to https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/getting-started/accessing-resources/getting-access-to-compute.md#monthly-budget.